Document change for `compact-full-write-cold-duration` (#3053)
influxdata/docs-v2: content/enterprise_influxdb/v1.9/administration/config-data-nodes.md

#### `compact-full-write-cold-duration = "4h"`

> The duration at which to compact all TSM and TSI files in a shard if they have not received a write or delete.

Author: Should this be "it", meaning the shard?

```suggestion
The duration at which to compact all TSM and TSI files in a shard if it has not received a write or delete.
```

Reviewer: Yes

Author: 👍
